server_environment_iap/models/iap_account.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Camptocamp SA <https://www.camptocamp.com/>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class IapAccount(models.Model):
+    _name = "iap.account"
+    _inherit = [
+        "iap.account",
+        "server.env.techname.mixin",
+        "server.env.mixin",
+    ]
+
+    name = fields.Char("Name", required=True)
```

Comment:
this field is not needed. The original model has `service_name` and the `tech_name` is the one used for env conf.

Reply:
probably I don't understand something, but without this field it doesn't work as I expect)
So, I need `conf` file to include such iap.account configuration:
```
[iap_account.resource_name]
service_name = my_service_name
account_token = my_secret_token
```
How should I use `tech_name` field then?
This code works like this: `name` becomes `tech_name` and unique, so then if I create new iap.account in UI `service_name` and `account_token` are filled with values from `conf`.
I can't use `service_name` as `tech_name` as due to source code iap accounts are company dependent, so we can have for example: partner_autocomplete in Company A and partner_autocomplete in Company B. Both service names are correct.
If you can point me where I'm wrong I would highly appreciate it. Thanks

Reply:
I got it. Sorry for the noise.